### DISCUSSION TITLE
Will there be a <Dynamic> equivalence?

### DISCUSSION THREAD

**Discussion by escwxyz** [leptos-rs/leptos#438] [General]
Something like [this](https://www.solidjs.com/docs/latest/api#dynamic)?

**Comment by gbj** [comment]
Hi — Sorry I missed this. I think the differences between the Rust type system and JavaScript type system make this harder in our model than Solid's, but if there's an API you can think of that you'd like to propose I'm happy to try to figure out how to do it.
